kiwi/xml_description.py

```diff
@@ -52,9 +53,9 @@ class XMLDescription(object):
         path to base XML description file
 
     """
-    def __init__(self, description, derived_from=None):
+    def __init__(self, description=None, derived_from=None, content=None):
         self.description_xslt_processed = NamedTemporaryFile()
-        self.description = description
+        self.description = content and BytesIO(content) or description
```

Comment:
Ah sure, is a matter of clarity. `and` and `or` was idiomatic like in 2.4 (10 years ago?) Still confusing the precedence of content over description for the value of self.description. This precedence is even more clear with the `if` expression